vmdk_plugin/drivers/vmdk/vmdk_driver.go

```diff
@@ -38,12 +38,7 @@ import (
 	"github.com/vmware/docker-volume-vsphere/vmdk_plugin/utils/refcount"
 )
 
-const (
-	devWaitTimeout   = 1 * time.Second
-	sleepBeforeMount = 1 * time.Second
-	watchPath        = "/dev/disk/by-path"
-	version          = "vSphere Volume Driver v0.4"
-)
+const version = "vSphere Volume Driver v0.4"
```

Comment:
nah. Just 0.5 will work . Otherwise we'd need to sync it with releases and it's not important at the moment, we can still match